### DISCUSSION TITLE
stable-fast is fast. really fast.

### DISCUSSION THREAD

**Discussion by shawnrushefsky** [chengzeyi/stable-fast#57] [Show and tell]
Just ran a [large benchmark](https://blog.salad.com/stable-diffusion-v1-5-benchmark/) comparing stable-fast against some other SD generation options, and it really did outstanding across the board on both performance and cost performance. Great job @chengzeyi !

**Comment by chengzeyi** [comment]
@shawnrushefsky Very impressive work! By the way, Have you tried my latest version v0.0.12.post6? It is about 5% faster than previous versions and it provides the most fastest cold start time! It only takes less than 10 seconds to compile on my machine with 4080!🚀

I spent a lot of time fine-tuning it in recent days. Now it is totally boosted!😄

**Reply by shawnrushefsky** [reply]
awesome! I haven't tried post6 yet. All that performance w/ only 10 seconds coldstart is super impressive
